OlawaleJoseph/micro-reddit
# Micro Reddit

This a project based on understanding the concepts of ActiveRecord in Ruby on Rails.

### Testing
- Get a local copy of this repo
  Open a terminal and run: ```git clone git@github.com:OlawaleJoseph/micro-reddit.git```
- Open a terminal and run ```bunle install```
- Run ```yarn install --check-files```
- Run ```rails c ```
- Create the following Entities to test
  - User (name: string)
  - Post (body: string)
  - Comment (body: string)
